docs/index.rst

```diff
@@ -24,23 +24,17 @@ Table of Contents
 =================
 
 .. toctree::
-    :maxdepth: 4
-    :numbered:
-
-    ptoled
-    keyboard
-    ptbuttons
-    pulse/eeprom
-    pulse/manual-configuration
-
-    input_components
-    output_components
-
-    component_examples
-    basic_examples
-    advanced_examples
-
-    parameters
+    :maxdepth: 2
```

Comment:
I have to say that I think we should try and group a lot of the later pages under a specific heading and open this up a bit. The main SDK page gets lost in and amongst all of the secondary pages..

Reply:
I am specifically referring to:
* faq
* contributing
* references
* license
* release_notes

Reply:
Agreed!